browser/ui/wallet_bubble_manager_delegate_impl.cc

```diff
@@ -6,8 +6,11 @@
 #include "brave/browser/ui/wallet_bubble_manager_delegate_impl.h"
 
 #include <utility>
+#include <vector>
 
+#include "base/callback.h"
 #include "brave/browser/ui/views/frame/brave_browser_view.h"
+#include "chrome/browser/extensions/extension_tab_util.h"
```

Comment:
we shouldn't be using extension code here and also this would require an extensions guard. `sessions::SessionTabHelper::IdForTab(web_contents)`

Reply:
test

Reply:
test

Reply:
test

Reply:
test

Reply:
moved t wallet_common_ui